media-libs/opus/opus-1.3.recipe

```diff
@@ -24,28 +24,28 @@ Supported features are:
 * Dynamically adjustable bitrate, audio bandwidth, and frame size
 * Good loss robustness and packet loss concealment (PLC)
 * Floating point and fixed-point implementation"
-HOMEPAGE="http://www.opus-codec.org"
+HOMEPAGE="https://www.opus-codec.org"
```

Comment:
Thanks @enzo1982 for your PR!

Could you please add a trailing slash to [`HOMEPAGE`](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#homepage)?
```diff
-HOMEPAGE="https://www.opus-codec.org"
+HOMEPAGE="https://www.opus-codec.org/"
```
Thanks!

Reply:
Done